.circleci/config.yml

```diff
@@ -28,11 +28,11 @@ workflows:
       - test:
           docker_image: circleci/node:10-browsers
       - test:
-          docker_image: circleci/node:12-browsers
+          docker_image: cimg/node:12.13
       - test:
-          docker_image: circleci/node:13-browsers
+          docker_image: cimg/node:14.21
       - test:
-          docker_image: circleci/node:14-browsers
+          docker_image: cimg/node:16.19
       - test:
-          docker_image: circleci/node:16-browsers
+          docker_image: cimg/node:18.9
```

Comment:
oh, I just checked the latest version for v18 is v18.15. 
https://nodejs.org/dist/latest-v18.x/
Is there some reason to choose 18.9?

Reply:
info: https://github.com/CircleCI-Public/cimg-node